pkg/app/app.go

```diff
@@ -150,7 +150,7 @@ func CreateApp() App {
 
 	// TODO: Let's take all these validations and delegate them to the client
 	// pkg.
-	if app.config, err = CreateConfig(app.args.ConfigFile); err != nil {
+	if app.config, err = CreateConfig(app.args.ConfigFile, app.args.RegistryAuthFile); err != nil {
```

Comment:
Like I mentioned above, I would prefer specifying the auth stuff as a pointer to some other location still in the main registry config.